Networking/Networking/Settings/Settings.swift

```diff
@@ -8,4 +8,14 @@ public struct Settings {
     /// UserAgent to be used for every Networking Request
     ///
     public static var userAgent = "WooCommerce iOS"
+
+    /// WordPress.com API Base URL
+    ///
+    public static var wordpressApiBaseURL: String {
+        if ProcessInfo.processInfo.arguments.contains("mocked-wpcom-api") {
+            return "http://localhost:8282/"
+        }
+
+        return "https://public-api.wordpress.com/"
+    }
```

Comment:
My concern with this is that the block is evaluated every time this property is accessed. 

Since we rely on a launch argument, I believe it's safe if it's only calculated once. 

```suggestion
    public static var wordpressApiBaseURL: String = {
        if ProcessInfo.processInfo.arguments.contains("mocked-wpcom-api") {
            return "http://localhost:8282/"
        }

        return "https://public-api.wordpress.com/"
    }()
```

Reply:
Agreed! Thanks for pointing that out. I updated the branch with this change.